docs/LinuxTool/tree.md

```diff
@@ -2,8 +2,153 @@
 
 ## 1. 基本功用
 
+tree是linux系统下十分常用的一个小工具
+
+其基本作用是用于以树状图的方式显示指定目录下存在的各级子目录和子文件
+
+只使用linux内建的ls指令虽然能够获取任意目录下子目录和子文件的信息，但是却不能够展示完整地展示多层次的目录、文件及之间的关系
+
+在阅览较为复杂的工程时，tree的功能就显得非常重要，能够帮助我们对整个工程各部分的文件的关系有直观清晰的认识
+
 ## 2. 安装方法
 
+1. 确认系统的Linux版本
+
+    1.请先确认所在的linux系统的版本是RedHat系列（包括RedHat,CentOS等）还是Debian系列（包括Debian,Ubuntu等）
+
+    2.如果不是很清楚，请打开系统命令行输入指令'lsb\_release -a'，根据打印出的信息进行确认。确认完之后前往下一步
```

Comment:
It seems to use normal line starts with `(1)` instead of oredered list nested in an ordered list will be clearer

Reply:
resolve in commit(9369f9e)